lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -439,6 +451,25 @@ class PatternStopContents extends Component<Props, State> {
             />
           </Col>
         </Row>
+        <Row>
+          <Col xs={12}>
+            <FormGroup
+              controlId='stopHeadsign'
+              bsSize='small'>
+              <ControlLabel
+                // TODO: internationalize this
+                title={'Headsign that overrides trip headsign between stops.'}
+                className='small'>Stop headsign</ControlLabel>
+              <FormControl
+                type='text'
+                // TODO: internationalize this
+                placeholder={'Outbound'}
```

Comment:
Nit: Is there are better way to convey that headsigns defined here are used in limited circumstances?